crates/chia-sdk-driver/src/primitives/datalayer/datastore_info.rs

```diff
@@ -0,0 +1,286 @@
+use crate::{
+    DelegationLayer, DelegationLayerArgs, DriverError, Layer, MerkleTree, NftStateLayer,
+    OracleLayer, SingletonLayer, SpendContext, WriterLayerArgs, DELEGATION_LAYER_PUZZLE_HASH,
+    DL_METADATA_UPDATER_PUZZLE_HASH,
+};
+use chia_protocol::{Bytes, Bytes32};
+use chia_puzzles::nft::NftStateLayerArgs;
+use clvm_traits::{ClvmDecoder, ClvmEncoder, FromClvm, FromClvmError, Raw, ToClvm, ToClvmError};
+use clvm_utils::{tree_hash, CurriedProgram, ToTreeHash, TreeHash};
+use num_bigint::BigInt;
+
+pub type StandardDataStoreLayers<M = DataStoreMetadata, I = DelegationLayer> =
+    SingletonLayer<NftStateLayer<M, I>>;
+
+#[derive(Debug, Clone, Copy, PartialEq, Eq, ToClvm, FromClvm)]
+#[repr(u8)]
+#[clvm(atom)]
+pub enum HintType {
+    // 0 skipped to prevent confusion with () which is also none (end of list)
+    AdminPuzzle = 1,
+    WriterPuzzle = 2,
+    OraclePuzzle = 3,
+}
+
+impl HintType {
+    pub fn value(&self) -> u8 {
```

Comment:
Can't you just do `hint_type as u8` already?

Reply:
yep